crates/primitives/src/stage/checkpoints.rs

```diff
@@ -17,6 +17,7 @@ pub struct MerkleCheckpoint {
     pub target_block: BlockNumber,
     /// The last hashed account key processed.
     pub last_account_key: B256,
+    // TODO: remove in the next breaking release.
     /// The last walker key processed.
     pub last_walker_key: Vec<u8>,
```

Comment:
Nothing i can do here atm without breaking the format, will open a separate PR for this